python/pfs/drp/stella/synthetic.py

```diff
@@ -292,12 +293,42 @@ def makeSyntheticPfsConfig(config, pfsDesignId, visit, rng=None,
 
     fiberStatus = np.full_like(targetType, FiberStatus.GOOD)
 
-    fiberMag = [np.array([22.0, 23.5, 25.0, 26.0] if
+    fbr_mag = [22.0, 23.5, 25.0, 26.0]
```

Comment:
Fix name `fbr_mag` (no need to drop two characters, and prefer camelCase to under_scores).